unlock-app/src/components/interface/connect/Wallet.tsx

```diff
@@ -45,11 +45,10 @@ export const ConnectViaEmail = ({
     setValue,
   } = useForm<UserDetails>()
 
-  if (email) {
-    setValue('email', email)
-  }
-
   const onSubmit = async (data: UserDetails) => {
+    if (email) {
```

Comment:
I am confused aboutthis. Why do we need to `setValue` in the form once it is submitted?